### DISCUSSION TITLE
Welcome to Honkai-Star-Rail-farm-script Discussions!

### DISCUSSION THREAD

**Discussion by zockerhuhn** [zockerhuhn/Honkai-Star-Rail-farm-script#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TUYENbro2** [comment]
Can you make a video tutorial? I don't know how to do it

**Reply by zockerhuhn** [reply]
I can do that, might take a bit but I'll notify you here when I have it
